apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.html

```diff
@@ -1,23 +1,41 @@
-<div (keyup)="onKeyInput($event)">
+<div (keyup)="onKeyInput($event)" class="vo-theme">
   <h1 class="page-subtitle">{{'VO_DETAIL.MEMBERS.TITLE' | translate}}</h1>
-  <button mat-flat-button color="accent" class="mr-2" (click)="onAddMember()">
-    {{'VO_DETAIL.MEMBERS.ADD_MEMBER' | translate}}
-  </button>
-  <button mat-flat-button color="warn" class="mr-2" [disabled]="selection.isEmpty()" (click)="onRemoveMembers()">
-    {{'VO_DETAIL.MEMBERS.REMOVE_MEMBERS' | translate}}
-  </button>
-  <button mat-stroked-button class="mr-2 mb-2" (click)="onListAll()">
-    {{'VO_DETAIL.MEMBERS.LIST_ALL' | translate}}
-  </button>
+  <div class="align-elements">
+    <button mat-flat-button color="accent" class="mr-2" (click)="onAddMember()">
+      {{'VO_DETAIL.MEMBERS.ADD_MEMBER' | translate}}
+    </button>
+    <button mat-flat-button color="warn" class="mr-2" [disabled]="selection.isEmpty()" (click)="onRemoveMembers()">
+      {{'VO_DETAIL.MEMBERS.REMOVE_MEMBERS' | translate}}
+    </button>
+    <button mat-stroked-button class="mr-2" (click)="onListAll()">
+      {{'VO_DETAIL.MEMBERS.LIST_ALL' | translate}}
```

Comment:
I would change this label, because right now it's bit confusing that you select some status to filter and you have to press button 'List all members'. Maybe change the button label dynamically in relation to selected status.